saleor/graphql/discount/mutations.py

```diff
@@ -33,6 +37,14 @@ class BaseDiscountCatalogueMutation(BaseMutation):
     class Meta:
         abstract = True
 
+    @classmethod
+    def recalculate_discounts(cls, products, categories, collections):
```

Comment:
It recalculates prices, not exactly discounts. Am I right? `discounts` associates with `Discount` model

Reply:
The recalculated `minimal price` includes discounts hence the name.
For clarity I've renamed it to `recalculate_minimal_price`.